Makefile

```diff
@@ -328,9 +328,12 @@ benchmark:
 
 lint:
 	golangci-lint run --out-format=tab
+	docker run --rm -i -v "$(CURDIR):/work" tmknom/markdownlint
```

Comment:
markdownlint is an easy dependency. It's an overhead to use `docker` for such a small task.  What do you think?

However, if we will end up using docker, then let's don't remove container and reuse it. Few weeks ago I created an example for it in `proto-gen` make job.

Reply:
Ok, I don't have a strong preference but I think docker would be preferred. I have removed the `--rm` flag.

Reply:
removing `--rm` flag is not a solution. Check the `proto-gen` job for a better solution.

Reply:
:facepalm: Not sure what I was thinking. I have updated based on the `proto-gen` command. Thank you.